test/api/workspaces/reset_sandbox.test.ts

```diff
@@ -9,6 +10,13 @@ test("POST /workspaces/reset_sandbox", async (t: ExecutionContext) => {
     db,
   } = await getTestServer(t)
 
+  const cs = db.client_sessions.find((cs) => cs.token === ws2.cst)
```

Comment:
Aren't only console sessions JWT? A client session cannot reset a sandbox workspace. What is this testing?